layers/cycle_paths.py

```diff
@@ -0,0 +1,149 @@
+from utils import *
+
+
+def makeCyclePaths(osm_input):
+    if not osm_input:
+        raise Exception("You must specify --osm_input")
+
+    filename = "cycle_paths"
+    tmp = f"tmp_{filename}"
+    ensureEmptyTempDirectoryExists(tmp)
+
+    run(
+        [
+            "osmium",
+            "tags-filter",
+            osm_input,
+            "w/highway",
+            "-o",
+            f"{tmp}/cycle_paths.osm.pbf",
+        ]
+    )
+    gjPath = f"{tmp}/cycle_paths.geojson"
+    convertPbfToGeoJson(
+        f"{tmp}/cycle_paths.osm.pbf", gjPath, "linestring", includeOsmID=True
+    )
+
+    cleanUpGeojson(
+        gjPath, getProps, filterFeatures=lambda f: getProps(f["properties"]) != None
+    )
+
+    convertGeoJsonToPmtiles(f"{tmp}/cycle_paths.geojson", "output/cycle_paths.pmtiles")
+
+
+# If this has some kind of cycle-path, returns a dictionary with:
+#
+# - kind = track | lane | shared_use_segregated | shared_use_unsegregated
```

Comment:
This is a vastly simplified schema. I bounced around a bunch of other ideas and this is where I landed.